app/perf_overlay.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_SYS_HEAP_RUNTIME_STATS=y
-CONFIG_TIMING_FUNCTIONS=y
\ No newline at end of file
+CONFIG_TIMING_FUNCTIONS=y
+CONFIG_DMA_INTEL_ADSP_HDA_TIMING_L1_EXIT=n
```

Comment:
what exactly is a performance build anyways? We already have debug/release, trace may or may not be enabled and now perf? seriously?

Reply:
@plbossart ，except for debug and release, we setup a new performance build to measure each module/task performance(mcps/cpc), it will add app/perf_overlay.conf config.
Performance build generate more performance logs compared with debug build.
These logs will be decoded with CI script: https://github.com/thesofproject/sof-test/blob/main/tools/sof_perf_analyzer.py

and in CI, there is daily performance build to check performance data and possible regression for each component:
Daily_performance: https://sof-ci.ostc.intel.com/#/result/planresultdetail/35118

Reply:
You are changing the definition of this perf_overlay.

It was "additional logs for performance analysis" and now it's become "L1 disabled".

That's not quite right, sorry. The additional logs are already questionable in that this creates Heisenbugs, but now even the behavior is different. 

Every time we create a new configuration we should ask ourselves if this really helps and what the additional validation work is v. benefits.

Reply:
it is still used for module performance analysis, this PR just want to make the measure more accurate, since our CPC is based max(peak), this PR can reduce peak 50% around for some modules.